auraed/src/init/power.rs

```diff
@@ -37,7 +37,9 @@ use ::libc;
 #[allow(dead_code)]
 pub(crate) fn syscall_reboot(action: i32) {
     unsafe {
-        libc::reboot(action);
+        if libc::reboot(action) != 0 {
+            panic!("failed to reboot");
```

Comment:
I don't think we want to panic. However, power.rs needs other work anyway. If you make other changes to this PR before merging, throw in a "TODO: Handle this better", just to make sure it doesn't get overlooked.